modules/navigation/nav_region.h

```diff
@@ -33,21 +33,13 @@
 
 #include "scene/resources/navigation_mesh.h"
 
-#include "nav_rid.h"
+#include "nav_base.h"
 #include "nav_utils.h"
 
-#include <vector>
-
-class NavMap;
-class NavRegion;
-
-class NavRegion : public NavRid {
+class NavRegion : public NavBase {
 	NavMap *map = nullptr;
 	Transform3D transform;
 	Ref<NavigationMesh> mesh;
-	uint32_t navigation_layers = 1;
-	float enter_cost = 0.0;
-	float travel_cost = 1.0;
 	Vector<gd::Edge::Connection> connections;
```

Comment:
Where do these enter and travel costs for the region go to in the new version? Or are they removed?

Reply:
They were moved to `NavBase`, a common base class that represents properties used as part of calculating the path in `get_path`.  Both `NavRegion` and `NavLink` share this base class since they act similarly.

It also removed the need to detect the type of `NavRegion` or `NavLink` in the `get_path` logic.